pubnub-users/src/integrationTest/kotlin/com/pubnub/user/UserIntegTest.kt

```diff
@@ -39,7 +38,8 @@ class UserIntegTest {
 
     @BeforeEach
     fun setUp() {
-        val config = PNConfiguration(UserId("kotlin")).apply {
+        UserId("kotlin")
```

Comment:
What is this line for?

Reply:
That's a leftover after failed attempt of mass fix